core/src/main/groovy/noe/tomcat/configure/envars/RpmTomcatEnvVarsFileFactory.groovy

```diff
@@ -73,6 +75,22 @@ class RpmTomcatEnvVarsFileFactory {
     }
   }
 
+  private static class Jws6RpmTomcatEnvVarsFile extends RpmTomcatEnvVarsFileBase {
+    Jws6RpmTomcatEnvVarsFile() {
+      Platform platform = new Platform()
+      if (platform.isRHEL8() || platform.isRHEL9() || platform.isRHEL10()) {
+        this.envFile = new File('/etc/opt/rh/scls/jws6/sysconfig/tomcat')
+      } else {
+        this.envFile = new File('/etc/opt/rh/jws6/sysconfig/tomcat')
+      }
```

Comment:
@ichristod It might be wise to use the new function OSVersionLessThan(...)  and invert the blocks, otherwise for each new release we'll have to keep adding them.